index.html

```diff
@@ -460,31 +460,28 @@ <h2>
         </p>
       </div>
       <p>
-        A <a>URL</a> <var>target</var> is said to be <dfn>within scope</dfn> of
-        <a>navigation scope</a> <var>scope</var> if the following algorithm
-        returns <code>true</code>:
+        A [=URL=] |target:URL| is said to be <dfn data-dfn-for="URL"
+        data-export="">within scope</dfn> of [=URL=] |scope:URL| if the
+        following algorithm returns `true`:
       </p>
-      <ol>
-        <li>Let <var>scopePath</var> be the [=string/concatenation=] of
-        <var>scopes</var>'s <a data-cite="URL#concept-url-path">path</a>, using
-        U+002F (/).
+      <ol class="algorithm">
+        <li>If |target| and |scope| are not [=same origin=], return `false`.
         </li>
-        <li>Let <var>targetPath</var> be the [=string/concatenation=] of <var>
-          target</var>'s <a data-cite="URL#concept-url-path">path</a>, using
-          U+002F (/).
+        <li>Let |scopePath:string| be the [=string/concatenation=] of
+        |scopes|'s [=URL/path=] using U+002F (/) as the separator.
         </li>
-        <li>If <var>target</var> is <a>same origin</a> as <var>scope</var> and
-        <var>targetPath</var> starts with <var>scopePath</var>, return
-        <code>true</code>.
+        <li>Let |targetPath:string| be the [=string/concatenation=] of
+        |target|'s [=URL/path=] using U+002F (/) as the separator.
         </li>
-        <li>Otherwise, return <code>false</code>.
+        <li>Return a [=boolean=] indicating if |targetPath| starts with
+        |scopePath:string|.
         </li>
       </ol>
       <p>
-        A <a>URL</a> <var>target</var> is said to be <dfn data-lt=
-        "within-scope-manifest">within scope of a manifest</dfn>
-        <var>manifest</var> if <var>target</var> is <a>within scope</a> of the
-        navigation scope of <var>manifest</var>.
+        A [=URL=] |target:URL| is <dfn data-export="" data-dfn-for=
+        "processed manifest" data-lt="within scope">within navigation scope of
```

Comment:
Sorry I just ... can't get my head around GitHub's commenting system. It is truly the worst code review system I have ever used (including email). I don't seem to be able to reply to a comment thread above, so starting a new one...

> Looks good, but I still would prefer to call it "within scope of a manifest" rather than "within the navigation scope of a processed manifest" (the "navigation scope" is implied, and the "processed" is explicit later).
>
> And I don't understand why you say "within scope of the processed manifest's scope member" when you could say "within scope of the navigation scope of the processed manifest"; now those are defined as equivalent, so you may as well use the term you defined.

You didn't address this comment. Do you disagree or did you just miss it?

Reply:
I addressed it. It's formally defined as:

 * "within navigation scope of a processed manifest".

However, when exported, referenced, and cited, you just do:

> Some |url| is [=processed manifest/within scope=] of a |processed manifest|.

or, for example:
 
> If |url| is [=processed manifest/within scope=] of |manifestX|, then do something...

Reply:
Ok, so, I'll go with the less fancy version... "within scope of a manifest".